ui/plugins/controller.js

```diff
@@ -88,7 +93,11 @@ treeherder.controller('PluginCtrl', [
                     $scope.eta_abs = Math.abs($scope.job.get_current_eta());
                     $scope.typical_eta = $scope.job.get_typical_eta();
                     $scope.jobRevision = ThResultSetStore.getSelectedJob($scope.repoName).job.revision
-
+                    $scope.job_ids = results[4];
+                    console.log(results);
+                    _.forEach($scope.job_ids, function(job) {
+                        console.log(job[0].value);
+                    });
```

Comment:
<img width="393" alt="screen shot 2015-10-31 at 11 37 10 am" src="https://cloud.githubusercontent.com/assets/2166227/10861667/c436b958-7fc3-11e5-9cef-159587412aea.png">
Hi Will, you can see the ts_paint and sessionrestore test missed in the picture which I think they shouldn't. And you can see I out put each value we got by job id:

> 1056, 244.77, 737.89, 1056, 1609.5, 2145, 798, 2145, 338, 798, 244.77

so their value haven't been found in the signature object list we retrieved by job id either.
@wlach